### ISSUE TITLE
Change example doc of heat template with F5 Plugins

### ISSUE THREAD

**Issue by pjbreaux** [F5Networks/f5-openstack-heat-plugins#50]
The example plugin in the doc is F5::Sys::iAppTemplate, but it should be F5::Sys::iAppCompositeTemplate.

**Comment by jputrino** [comment]
@pjbreaux Is this still a valid issue?

**Comment by pjbreaux** [comment]
@jputrino: yes it is still an issue. The plugin referenced in that doc does not exist. The one that is expected there is iAppFullTemplate.
